Shape.cs

```diff
@@ -0,0 +1,23 @@
+
+
+
+using System.Data;
+
+abstract class Shape
+{
+    public bool FillShape {get; set;}
```

Comment:
The boolean variable's name should be like a yes/no question. For e.g. `IsFilled`